helm/templates/raw-spans-grouper-config.yaml

```diff
@@ -6,42 +6,30 @@ metadata:
     release: {{ .Release.Name }}
 data:
   application.conf: |-
-    flink.job {
-      {{- if hasKey .Values.rawSpansGrouperConfig.flink "job" }}
-      {{- if hasKey .Values.rawSpansGrouperConfig.flink.job "parallelism" }}
-      parallelism = {{ int .Values.rawSpansGrouperConfig.flink.job.parallelism }}
-      {{- end }}
-      {{- if hasKey .Values.rawSpansGrouperConfig.flink.job "metrics" }}
-      metrics {
-        metrics {
-          reporters = {{- toJson .Values.rawSpansGrouperConfig.flink.job.metrics.reporters | trim | nindent 12 }}
-        }
-      }
-      {{- end }}
-      {{- end }}
-    }
 
-    flink.source {
-      schema.registry {
-        schema.registry.url = "{{ .Values.rawSpansGrouperConfig.flink.source.schema.registry.url }}"
-        specific.avro.reader = true
-      }
-      kafka {
-        bootstrap.servers = "{{ .Values.rawSpansGrouperConfig.flink.source.kafka.bootstrapServers }}"
-      }
-    }
+    kafka.streams.config = {
+      application.id = raw-spans-to-structured-traces-grouping-job
+      state.dir = "/var/data/"
+      metrics.recording.level = "{{ .Values.rawSpansGrouperConfig.kafka.streams.config.metricsRecordingLevel }}"
+      num.stream.threads = "{{ .Values.rawSpansGrouperConfig.kafka.streams.config.numStreamThreads }}"
+      num.standby.replicas=1
+      replication.factor = 3
+      producer.compression.type = gzip
+      producer.max.request.size = 10485760
+      topic.cleanup.policy = "delete,compact"
+      topology.optimization = all
+      bootstrap.servers = "{{ .Values.rawSpansGrouperConfig.kafka.streams.config.bootstrapServers }}"
+      auto.offset.reset = "latest"
+      auto.commit.interval.ms = 5000
```

Comment:
`auto.commit.interval.ms = 5000` and `auto.offset.reset = "latest"` are also present in base defaults.

Reply:
fixed